src/states/node/mod.rs

```diff
@@ -1538,7 +1538,12 @@ impl Node {
         sent_to: BTreeSet<XorName>,
     ) -> Result<(), RoutingError> {
         if !self.crust_service().is_connected(target) {
-            trace!("{} Not connected to {:?}. Dropping peer.", self, target);
+            trace!(
+                "{} Not connected to {:?}. Dropping peer and message: {:?}",
+                self,
+                target,
+                &signed_msg
```

Comment:
wasnt just `signed_msg.routing_message()` enough here if we wanted msg details, or were you interested in things to do with proving_sections/... they normally tend to be the large part of messages for log spam.

Reply:
No I was only interested in the first part, will update.